### ISSUE TITLE
Order a pile or the entire deck

### ISSUE THREAD

**Issue by brneor** [crobertsbmw/deckofcards#89]
Is there currently a method to order the cards? This is a nice to have feature.

**Comment by crobertsbmw** [comment]
If you open a new deck and don't shuffle the cards, then they will be in
order, but that's about it.

**Comment by johnnycopes** [comment]
Agreed that it would be really helpful for debugging purposes. The use case I've come across is that it'd be useful create a deck of select cards in a specific order in order to write tests for particular outcomes

**Comment by roiko20** [comment]
I agree as well, it will be a great feature.
@crobertsbmw if I create a partial deck and the order of cards is important for the logic of my app, when I call:
`https://deckofcardsapi.com/api/deck/new/?cards=2C,8H,3D,JC,9S,4S,7C,2D,0C,QH,3H,6D,8C,AH,0D,KC,7H,KD,9C,3S,8D`
for example, which should return unshuffled as `/shuffle/` is not specified, when I draw cards from it, it does not return in the order I specified.
It always returns ordered, I think it's Spades->Diamonds->Clubs->Hearts, so in the above example, I always get 3S first.
However, getting 2C,8H,3D... as in the order I specified in the new deck request is important for me.
Really appreciate it if you can help me somehow 👍 
This is a great project, really appreciate it!

**Comment by crobertsbmw** [comment]
Okay, I think this is fixed now. Thanks!
